financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsPlaygroundViewModel.kt

```diff
@@ -44,15 +44,15 @@ class FinancialConnectionsPlaygroundViewModel(
         when (flow) {
             Flow.Data -> startForData(mode)
             Flow.Token -> startForToken(mode)
-            Flow.PaymentIntent -> startWithPaymentIntent()
+            Flow.PaymentIntent -> startWithPaymentIntent(mode)
```

Comment:
We were ignoring the test mode on PaymentIntent flows.

Reply:
good find